installer/pkg/broker-cli/client/adapter/httpadapter.go

```diff
@@ -39,6 +40,32 @@ type CreateBrokerParams struct {
 	Title   string
 }
 
+// GetCatalogParams is used as input to GetCatalog.
+type GetCatalogParams struct {
+	// Server is URL of the broker.
+	Server string
```

Comment:
consider URL to match CreateBrokerParams

Reply:
done